src/test/suite/fixtures/recorded/decorations/moveLineAir.yml

```diff
@@ -0,0 +1,53 @@
+languageId: plaintext
+command:
+  spokenForm: move line air
+  version: 2
+  targets:
+    - type: primitive
+      mark: {type: decoratedSymbol, symbolColor: default, character: a}
+      modifiers:
+        - type: containingScope
+          scopeType: {type: line}
+    - {type: primitive, isImplicit: true}
+  usePrePhraseSnapshot: true
+  action: {name: moveToTarget}
+initialState:
+  documentContents: |
+    a a a
+    b b b
+    c c c
+  selections:
+    - anchor: {line: 3, character: 0}
+      active: {line: 3, character: 0}
+  marks:
+    default.a:
+      start: {line: 0, character: 0}
+      end: {line: 0, character: 1}
+finalState:
+  documentContents: |-
+    b b b
+    c c c
+    a a a
+  selections:
+    - anchor: {line: 2, character: 5}
+      active: {line: 2, character: 5}
+  thatMark:
+    - anchor: {line: 2, character: 0}
+      active: {line: 2, character: 5}
+  sourceMark:
+    - anchor: {line: 0, character: 0}
+      active: {line: 0, character: 0}
+decorations:
+  - name: pendingDeleteBackground
+    type: line
+    start: {line: 0, character: 0}
+    end: {line: 0, character: 5}
+  - name: pendingModification0Background
+    type: token
+    start: {line: 3, character: 0}
+    end: {line: 3, character: 0}
```

Comment:
Any idea where this is coming from?

Reply:
Yes that is my current selection. ie the destination we are replacing.

Reply:
ohh interesting got it